### PR TITLE
Cleanup node id creation in shim, organize env

### PR DESCRIPTION
**Pull request by DiegoRBaquero**
- Environment variables cleanup and organization
- Per node hash using a per version hash

### CODE REVIEW

File: container/start.sh [hunk 1]
```diff
@@ -51,6 +51,8 @@ fi
 
 nginx
 
+export HASH=$(echo -n "$VERSION_HASH$(cat /usr/src/app/shared/nodeId.txt)" | sha256sum | head -c 64)
```
Comment (guanzo) [inline container/start.sh:54]: Keep it as `VERSION_HASH`? Easier if the variable name is constant.

Comment (guanzo) [inline container/start.sh:54]: Think `HASH` needs to be a little more specific. Like `NODE_VERSION_HASH`

File: scripts/run.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ mkdir -p "$(pwd)/shared"
 echo "$(date -u) [host] Running Saturn node dev, with volume in $(pwd)/shared"
 
 # Start the docker image
-docker run --name saturn-node --rm \
+docker run --name saturn-node -it --rm \
```
[the remark this answers is not in the document]
Comment (guanzo) [reply scripts/run.sh]: In the meantime i've been using `docker rm -f saturn-node` to stop the container 🥴

[the remark this answers is not in the document]
Comment (DiegoRBaquero) [reply scripts/run.sh]: I use `docker stop -t 1 saturn-node` to at least allow it to deregister

**Pull request merged**